modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -202,6 +224,10 @@ def fetch_appointment_requests
           }
         end
 
+        def format_facility_ids(ids)
```

Comment:
Given that the inputs here come from appointments.map, it must be an array. I don't think you need handling for other types.

Reply:
got it. I changed it to join as a string

Reply:
nope it doesn't so I removed it